### DISCUSSION TITLE
What is the meaning behind "id='~_XYZ'" ?

### DISCUSSION THREAD

**Discussion by shakazulu89** [serenity-is/Serenity#7236] [General]
hey

can anyone explain/remind me whats the point of setting element ids prefixed with "~_"? such as:
<div id="~_OrdersGrid">

I cant find any info about this. I feel like it had something to do with preventing multiple instances of the same element having the same ID attribute, but im not sure.

Thanks!

**Comment by VictorTomaili** [comment]
it is auto generated idprefix placeholder. you can access it with `this.idPrefix` property. When using tsx file you should use `const id = this.useIdPrefix();` and `id.MyEditorElement` to generate auto id. Use `this.byId("MyEditorElement")` to get element.
